modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/LayoutScreen/ModalAddObjectLayoutTab.tsx

```diff
@@ -270,15 +294,33 @@ const ModalAddObjectLayoutTab: React.FC<IModalAddObjectLayoutTabProps> = ({
 								] ?? selectedRelationship?.name
 							}
 						>
-							{({label, name, type}) => (
+							{({label, name, reverse, type}) => (
 								<div className="d-flex justify-content-between">
 									<div>
 										{label[defaultLanguageId] ?? name}
 									</div>
 
 									<div className="object-web-relationship-item-label">
-										<ClayLabel displayType="secondary">
-											{separateCamelCase(type)}
+										<ClayLabel
+											displayType={
```

Comment:
Hey @naraandrade, Thanks for the review, to make your suggestion I had to create a `getRelationshipInfo` function taking the `reverse` and `type` as parameters outside the `ModalAddObjectLayoutTab` component to avoid unnecessary rendering, the reason for this is because the `AutoComplete` component receives as a child each item in the list we pass to it, so we have two different situations, one above with the declared value `selectedRelationshipInfo`, and the `children` of the `AutoComplete` component which is basically an interaction with different values, so I think creating a function for that is the best solution, what do you think?